integration-tests/sanity/sanity_test.go

```diff
@@ -23,7 +23,7 @@ var _ = Describe("Sanity", func() {
 	Context("should update storageclass", func() {
 		It("should update storageclass wih replica count to 1", func() {
 			// Fetching the openebs component artifacts
-			artifacts, err := artifacts.GetArtifactsListUnstructured(artifacts.SingleReplicaSC)
+			artifacts, err := artifacts.GetArtifactsListUnstructuredFromFile(artifacts.SingleReplicaSC)
```

Comment:
Mark a TODO as these things should not be in artifacts yaml.
Only the openebs operator contents will reside in artifacts yaml

Reply:
added